tests/minikube.sh

```diff
@@ -127,72 +157,84 @@ kadalu_operator)
 
 test_kadalu)
     fail=0
+    date
     echo "Requesting PVC and Sample apps"
+    echo "Running default sample test app yml from repo"
     cp examples/sample-test-app.yaml /tmp/kadalu-test-app.yaml
     # Run ReadWriteOnce test
     kubectl create -f /tmp/kadalu-test-app.yaml
 
     # give it some time
     cnt=0
     while true; do
-        cnt=$((cnt+1))
-        sleep 1;
-        ret=`kubectl get pods pod1 | grep Completed | wc -l`
+        cnt=$((cnt + 1))
+        sleep 1
+        ret=$(kubectl get pods pod1 | grep 'Completed' | wc -l)
         if [[ $ret -eq 1 ]]; then
             echo "Successful after $cnt seconds"
-            break;
+            break
         fi
-        if [[ $cnt -eq 100 ]]; then
-            kubectl get pods -nkadalu;
-            kubectl get pods;
+        if [[ $cnt -eq 500 ]]; then
+	    kubectl get pvc
+            kubectl get pods -nkadalu
+            kubectl get pods
             echo "exiting after 100 seconds"
             fail=1
-            break;
+            break
         fi
-        if [[ $((cnt%10)) -eq 0 ]]; then
+        if [[ $((cnt % 25)) -eq 0 ]]; then
             echo "$cnt: Waiting for pods to come up..."
         fi
     done
-    kubectl logs pod1
 
     sed -i -e "s/pv1/pv2/g" /tmp/kadalu-test-app.yaml
     sed -i -e "s/pod1/pod2/g" /tmp/kadalu-test-app.yaml
     sed -i -e "s/ReadWriteOnce/ReadWriteMany/g" /tmp/kadalu-test-app.yaml
     kubectl create -f /tmp/kadalu-test-app.yaml
+    echo "Running sample test app yml from repo (with Type changed to RWX from RWO)"
     # give it some time
     cnt=0
     while true; do
-        cnt=$((cnt+1))
-        sleep 1;
-        ret=`kubectl get pods pod2 | grep Completed | wc -l`
+        cnt=$((cnt + 1))
+        sleep 1
+        ret=$(kubectl get pods pod2 | grep 'Completed' | wc -l)
         if [[ $ret -eq 1 ]]; then
             echo "Successful after $cnt seconds"
-            break;
+            break
         fi
         if [[ $cnt -eq 100 ]]; then
             kubectl get pods -nkadalu;
             kubectl get pods;
             echo "exiting after 100 seconds"
             fail=1
-            break;
+            break
         fi
-        if [[ $((cnt%10)) -eq 0 ]]; then
+        if [[ $((cnt % 10)) -eq 0 ]]; then
             echo "$cnt: Waiting for pods to come up..."
         fi
     done
-    kubectl logs pod2
 
     # Log everything so we are sure if things are as expected
     for p in $(kubectl -n kadalu get pods -o name); do
-        echo "====================== Start $p ======================";
```

Comment:
i think we can enable logging only if test fails

Reply:
I wanted log there, so in worst case, we can validate if things are OK or not OK just by looking at even successful test.

Anyways, if the tests are passing, we wouldn't bother to check the tests and would look at only code changes. Only when it fails, we goto CI/CD system. I am inclined to keep it.